scripts/send_digest.py

```diff
@@ -0,0 +1,112 @@
+"""Script for sending OSF email digests to subscribed users and removing the records once sent."""
+
+import logging
+import datetime
+from bson.code import Code
+from framework.tasks import app as celery_app
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound
+from framework import sentry
+from framework.auth.core import User
+from framework.mongo import database as db
+from website import mails
+from website.util import web_url_for
+from website.app import init_app
+from website.notifications.model import DigestNotification
+from website.notifications.utils import NotificationsDict
+
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Oh, good point! I was using this for debugging, but I don't think we need to see the output of every email sent in the logs.